packages/Algernon.package/ALGTopCategorizer.class/instance/categorize.withExistingCategories..st

```diff
@@ -0,0 +1,9 @@
+categorizing
+categorize: anItemList withExistingCategories: aCategoriesCollection 
```

Comment:
We just don't need it, as the TopCategories are generated independently from other categorizers.

I could have overwritten the `categorize:` Method, but then I would have to call it from the `categorize:withExistingCategories:` Method, which would invert the "flow" of the super class. Therefore I thought it would be acceptable to just have an unused argument.

Reply:
Yeah, I thought that, but wanted to check in.